build.gradle.kts

```diff
@@ -14,7 +14,8 @@ buildscript {
     val repos = listOfNotNull(
             bootstrapKotlinRepo,
             "https://jcenter.bintray.com/",
-            "https://plugins.gradle.org/m2")
+            "https://plugins.gradle.org/m2",
+            "https://oss.sonatype.org/content/repositories/snapshots")
```

Comment:
Please, add the snapshots repo directly to the `kotlin-script-util` build script. I think we don't want (yet) to add it to all projects.